{"name": "Login to OrangeHRM with different parameter -- @1.2 ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: disconnected: Unable to receive message from renderer\n  (failed to check if window was closed: disconnected: not connected to DevTools)\n  (Session info: chrome=114.0.5735.106)\nStacktrace:\n0   chromedriver                        0x0000000102d43f48 chromedriver + 4226888\n1   chromedriver                        0x0000000102d3c4f4 chromedriver + 4195572\n2   chromedriver                        0x0000000102980d68 chromedriver + 281960\n3   chromedriver                        0x000000010296a9b8 chromedriver + 190904\n4   chromedriver                        0x000000010296a6e0 chromedriver + 190176\n5   chromedriver                        0x0000000102969d1c chromedriver + 187676\n6   chromedriver                        0x000000010298ad54 chromedriver + 322900\n7   chromedriver                        0x00000001029f2e30 chromedriver + 749104\n8   chromedriver                        0x00000001029aff1c chromedriver + 474908\n9   chromedriver                        0x00000001029b0ef4 chromedriver + 478964\n10  chromedriver                        0x0000000102d0559c chromedriver + 3970460\n11  chromedriver                        0x0000000102d096f0 chromedriver + 3987184\n12  chromedriver                        0x0000000102d0f5b4 chromedriver + 4011444\n13  chromedriver                        0x0000000102d0a2fc chromedriver + 3990268\n14  chromedriver                        0x0000000102ce21c0 chromedriver + 3826112\n15  chromedriver                        0x0000000102d26088 chromedriver + 4104328\n16  chromedriver                        0x0000000102d261e0 chromedriver + 4104672\n17  chromedriver                        0x0000000102d35f28 chromedriver + 4169512\n18  libsystem_pthread.dylib             0x000000019a98bfa8 _pthread_start + 148\n19  libsystem_pthread.dylib             0x000000019a986da0 thread_start + 8\n\n", "trace": "  File \"/Library/Python/3.9/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Library/Python/3.9/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/orange_login_2.py\", line 25, in clickLogin\n    context.driver.find_element(By.XPATH, \"//button[contains(@type,'submit')]\").click()\n  File \"/Library/Python/3.9/site-packages/selenium/webdriver/remote/webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"/Library/Python/3.9/site-packages/selenium/webdriver/remote/webelement.py\", line 403, in _execute\n    return self._parent.execute(command, params)\n  File \"/Library/Python/3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"/Library/Python/3.9/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given launch chrome browser", "status": "passed", "start": 1686204165554, "stop": 1686204167213}, {"name": "When open OrangeHRM Home page", "status": "passed", "start": 1686204167213, "stop": 1686204177230}, {"name": "And Enter username \"adm\" and password \"admin123\"", "status": "passed", "start": 1686204177231, "stop": 1686204177412}, {"name": "And Click on login button", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: disconnected: Unable to receive message from renderer\n  (failed to check if window was closed: disconnected: not connected to DevTools)\n  (Session info: chrome=114.0.5735.106)\nStacktrace:\n0   chromedriver                        0x0000000102d43f48 chromedriver + 4226888\n1   chromedriver                        0x0000000102d3c4f4 chromedriver + 4195572\n2   chromedriver                        0x0000000102980d68 chromedriver + 281960\n3   chromedriver                        0x000000010296a9b8 chromedriver + 190904\n4   chromedriver                        0x000000010296a6e0 chromedriver + 190176\n5   chromedriver                        0x0000000102969d1c chromedriver + 187676\n6   chromedriver                        0x000000010298ad54 chromedriver + 322900\n7   chromedriver                        0x00000001029f2e30 chromedriver + 749104\n8   chromedriver                        0x00000001029aff1c chromedriver + 474908\n9   chromedriver                        0x00000001029b0ef4 chromedriver + 478964\n10  chromedriver                        0x0000000102d0559c chromedriver + 3970460\n11  chromedriver                        0x0000000102d096f0 chromedriver + 3987184\n12  chromedriver                        0x0000000102d0f5b4 chromedriver + 4011444\n13  chromedriver                        0x0000000102d0a2fc chromedriver + 3990268\n14  chromedriver                        0x0000000102ce21c0 chromedriver + 3826112\n15  chromedriver                        0x0000000102d26088 chromedriver + 4104328\n16  chromedriver                        0x0000000102d261e0 chromedriver + 4104672\n17  chromedriver                        0x0000000102d35f28 chromedriver + 4169512\n18  libsystem_pthread.dylib             0x000000019a98bfa8 _pthread_start + 148\n19  libsystem_pthread.dylib             0x000000019a986da0 thread_start + 8\n\n", "trace": "  File \"/Library/Python/3.9/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Library/Python/3.9/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/orange_login_2.py\", line 25, in clickLogin\n    context.driver.find_element(By.XPATH, \"//button[contains(@type,'submit')]\").click()\n  File \"/Library/Python/3.9/site-packages/selenium/webdriver/remote/webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"/Library/Python/3.9/site-packages/selenium/webdriver/remote/webelement.py\", line 403, in _execute\n    return self._parent.execute(command, params)\n  File \"/Library/Python/3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"/Library/Python/3.9/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1686204177412, "stop": 1686204178225}, {"name": "Then User must successfully login to the dashboard page", "status": "skipped", "start": 1686204178226, "stop": 1686204178226}], "parameters": [{"name": "username", "value": "adm"}, {"name": "password", "value": "admin123"}], "start": 1686204165553, "stop": 1686204178226, "uuid": "837c2d1f-f1ed-47fa-aab5-a0f93356e1f6", "historyId": "206f0999fff5561a6ba023f7a021bc7b", "fullName": "OrangeHRm Login: Login to OrangeHRM with different parameter", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "OrangeHRm Login"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}